.github/workflows/scala-unit-test.yml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2023 KU Leuven.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Xiaoling Yi <xiaoling.yi@esat.kuleuven.be>
+
+# Run Scala Unit Test
+name: Unit Test
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+
+jobs:
+  run:
+    name: Build and Test
+    runs-on: ubuntu-22.04
+    container:
+      image: ghcr.io/kuleuven-micas/snax:pr-96
```

Comment:
Please don't point to a random PR tag. This should work with ghcr.io/kuleuven-micas/snax:main or it should trigger an error.